packages/icons/scripts/build.js

```diff
@@ -0,0 +1,93 @@
+const upperFirst = require('lodash/upperFirst');
+const camelCase = require('lodash/camelCase');
+const snakeCase = require('lodash/snakeCase');
+const fs = require('fs');
+const path = require('path');
+const chalk = require('chalk');
+const SVGO = require('svgo');
+const svgr = require('@svgr/core');
+const svgoConfig = require('../svgo.config.js');
+
+const SVGS_DIR = path.resolve(__dirname, '../src');
+const LIB_DIR = path.resolve(__dirname, '../lib');
+
+const getTree = (source) => {
+  const sizes = fs.readdirSync(source).filter((size) => {
+    const stats = fs.statSync(path.resolve(`${source}/${size}`));
+    return stats.isDirectory();
+  });
+
+  const tree = sizes.reduce((acc, size) => ({
+    ...acc,
+    [size]: fs
+      .readdirSync(path.resolve(`${source}/${size}`))
+      // Important not to remap names to paths here
+      // Webpack won't be able to require full paths dynamically
+      .filter((name) => {
+        const filePath = path.resolve(`${source}/${size}/${name}`);
+        const stats = fs.statSync(filePath);
+        return !stats.isDirectory() && /\.svg$/.test(name);
+      }),
+  }), {});
+
+  return tree;
+};
+
+const getFiles = (tree) => (
+  Object.keys(tree)
+    .reduce((acc, size) => (
+      acc.concat(tree[size].map((name) => (
+        `${size}/${name}`
+      )))
+    ), [])
+);
```

Comment:
copied from `import_statements.jsx` - suggestion where to extract this stuff to?